test/typescript-karma/Data/Modifiers/MultiColumnSort.test.js

```diff
@@ -0,0 +1,43 @@
+import DataTable from '/base/code/es-modules/Data/DataTable.js';
+import SortModifier from '/base/code/es-modules/Data/Modifiers/SortModifier.js';
+
+QUnit.test('SortModifier.modify with multi-column', (assert) => {
```

Comment:
It would be good to test the arrays returned by `dataTable.getOriginalRowIndex` and `dataTable.getLocalRowIndex` here as well. The idea of these methods is to map the same rows in the original and the modified table. Check `DataTable.test.js` for the reference.